zebra-network/src/peer_set/initialize.rs

```diff
@@ -109,8 +111,54 @@ where
 {
     let (tcp_listener, listen_addr) = open_listener(&config.clone()).await;
 
-    let (address_book, address_book_updater, address_metrics, address_book_updater_guard) =
-        AddressBookUpdater::spawn(&config, listen_addr);
+    let (
+        address_book,
+        bans_receiver,
+        address_book_updater,
+        address_metrics,
+        address_book_updater_guard,
+    ) = AddressBookUpdater::spawn(&config, listen_addr);
+
+    let (misbehavior_tx, mut misbehavior_rx) = mpsc::channel(
+        3 * config
```

Comment:
Can we explain why the `3` is used ? Maybe by moving it to a constant or just with a comment.

Reply:
It's actually not necessary, I added a comment explaining why the peerset limit is used in https://github.com/ZcashFoundation/zebra/pull/9201/commits/88e0c9c0405ee533f6308394928a3f8de9f161d0.